LibreNMS/Interfaces/Polling/PrinterPolling.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * PrinterPolling.php
+ *
+ * Printer Polling interface
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @link https://www.librenms.org
+ */
+
+namespace LibreNMS\Interfaces\Polling;
+
+interface PrinterPolling
+{
+    /**
+     * @return
+     */
+    public function pollPrinter();
```

Comment:
should send it the models that need to be polled.

Reply:
These files are actually left-overs from the none implementation of os-overrides, should I implement it or delete them? :)

Reply:
ah, go ahead and delete.  Easy to re-add if needed.